templates/boznar/EXO_complaint.txt

```diff
@@ -0,0 +1,21 @@
+[small][center][Logo]
+Corporate Complaint Form C-2202[/center]
+
+[b]EXO Personnel Complaint Form[/b]
+Vessel: SEV Torch
+DTG: [date], [time]
+
+[b]Complainant:[/b] [field]
+[b]Complainee(s):[/b] [field]
+
+Complaint Overview:
+[field]
+
+Complaint Details:
+[field]
+
+[hr]
+Complainant's Signature: [field]
+Witness Signature: [field]
```

Comment:
Complainant's is possessive, Witness is not - let's do Complainant and Witness instead